docs/Configuration.md

```diff
@@ -375,29 +374,64 @@ Default `"off"`
 This feature automatically enables the --diff feature.
 
 ## Fallback version
-When enabling the --dashboard-compare feature you can provide a fallback version. This version will be used to download a report when we cannot find one for your current source version.  If you don't specify a value for fallback version, we will use --git-source as the fallback version.When we are unable to find a fallback version we will do a complete mutation testrun. 
-
-For example:
-You use a development branch and you use feature branches that merge back to development. Your fallback version would be development. 
-You start a pull request to merge your finished feature back into development. Since this is a new feature and a new pull request, you will not yet have a report for your feature branch and the fallback version is used instead.
-Since your feature branch is branched off of development, a report for development will be fairly up-to-date with your feature branch, and you will only have to test the new mutations added in your feature branch. All other mutants can be skipped, because we have an up-to-date report for the development branch.
+When enabling the --dashboard-compare feature you can provide a fallback version. This version will be used to pull a baseline when we cannot find a baseline for your current branch. When we are still unable to provide a baseline we will start a complete testrun to create a complete baseline.
 
 ```
 dotnet stryker --dashboard-compare --dashboard-fallback-version master
 dotnet stryker -compare -fallback-version master
 ```
-Default: `--git-source`
+Default: `null`
+
+## Baseline Storage location
+Sets the storage location for the baseline used by --dashboard-compare. By default this is set to disk, when the dashboard reporter is enabled this is automatically set to Stryker Dashboard.
+
+Supported storage locations are:
+
+| Storage location | Option | Description |
+|------------------|--------|-------------|
+| Disk             | Disk   | Saves the baseline to the `StrykerOutput` folder|
+| Stryker Dashboard| Dashboard | Saves the baseline to Stryker Dashboard |
+| Azure File Storage | AzureFileStorage | Saves the baseline to Azure File Storage |
+
+```
+dotnet stryker --dashboard-compare --baseline-storage-location disk
+dotnet stryker -compare -bsl disk
+```
+Defaut `"disk"`
+
+## Configuring Azure File Storage
+When using Azure File Storage as baseline storage location you are required to provide the following values.
+
+### Azure File Storage URL
```

Comment:
This is missing the azure file storage url parameters

Reply:
Please explain further. What url parameters?

Reply:
What commandline flag or json key should I set for the fileshare url? The docs only show an example of what the url looks like, not how to set it.